pkg/handler/metadata/metadata.go

```diff
@@ -62,6 +62,36 @@ func (h *handler) WorkingStatus(c *gin.Context) {
 	c.JSON(http.StatusOK, view.CreateResponse[any](res, nil, nil, nil))
 }
 
+// TechStacks godoc
+// @Summary Get list values for tech stacks
+// @Description Get list values for tech stacks
+// @Tags Metadata
+// @Accept  json
+// @Produce  json
+// @Success 200 {object} view.TechStackResponse
+// @Failure 400 {object} view.ErrorResponse
+// @Failure 500 {object} view.ErrorResponse
+// @Router /metadata/tech-stacks [get]
+func (h *handler) TechStacks(c *gin.Context) {
+	// 1 prepare the logger
+	// TODO: can we move this to middleware ?
```

Comment:
remove comment if it's not necessary

Reply:
remove this one `// 1 prepare the logger` and keep the TODO log

Reply:
dạ ok anh